zetta_utils/geometry/bbox.py

```diff
@@ -428,7 +428,7 @@ def pformat(self, resolution: Optional[Sequence[float]] = None) -> str:  # pragm
             f"({s.join([str(slice.stop) for slice in slices])})"
         )
 
-    def get_size(self) -> int | float:  # pragma: no cover
+    def get_size(self) -> Union[int, float]:  # pragma: no cover
```

Comment:
Is this change for readability or is there something broken with | syntax?

Reply:
This is actually what was breaking the integration tests. Some typeguard dependency must have updated, so it started breaking on this even though we have its version pinned